bgpd/bgp_route.c

```diff
@@ -11507,6 +11545,23 @@ static void bgp_show_path_info(struct prefix_rd *pfx_rd,
 	}
 }
 
+/*
+ * Return rd based on safi
+ */
+static const struct prefix_rd *rd_from_dest(const struct bgp_dest *dest,
```

Comment:
nit: Can be prefixed with `bgp_` here? Just to distinguish easily...

Reply:
done